source/agora/consensus/EnrollmentManager.d

```diff
@@ -489,6 +489,50 @@ public class EnrollmentManager
         return makeEnrollment(kp, utxo, cycle_length, cache[$ - offset - 1], offset);
     }
 
+    version (unittest)
+    private static void checkGenesisEnrollments(const Block genesisBlock, KeyPair[] keys, uint cycle_length)
+    {
+        import agora.consensus.data.Transaction : Transaction, TxType;
+        import std.format;
+        import std.range;
+        import std.conv;
+        import std.algorithm;
+
+        static struct NodeEnrollment
+        {
+            Enrollment enrol;
+            PublicKey key;
+        }
+        Hash txhash = hashFull(genesisBlock.txs.filter!(tx => tx.type == TxType.Freeze).front);
+        Hash[] utxos = iota(6).map!(i => UTXO.getHash(txhash, i)).array;
```

Comment:
To make slightly less assumptions, you could have an `enrollable` function similar to https://github.com/bpfkorea/agora/blob/e13aa77a2794df35d724344852960e02953b8c9d/source/agora/utils/Test.d#L646-L652

But with UTXOs instead of `TxBuilder`

Reply:
It is not really `enrollable` as these helpers are just returning either the frozen or payment tx from Genesis. I could improve on the naming though.

Reply:
I'll add `frozens` and `payments` as block functions.